gridfinity-rebuilt-bins.scad

```diff
@@ -73,11 +87,35 @@ div_base_y = 0;
 color("tomato") {
 gridfinityInit(gridx, gridy, height(gridz, gridz_define, style_lip, enable_zsnap), height_internal) {
 
-    if (divx > 0 && divy > 0)
-    cutEqual(n_divx = divx, n_divy = divy, style_tab = style_tab, scoop_weight = scoop);
+    if (divx > 0 && divy > 0) {
+        
+        cutEqual(n_divx = divx, n_divy = divy, style_tab = style_tab, scoop_weight = scoop);
+    
+    } else if (cdivx > 0 && cdivy > 0) {
```

Comment:
Move logic to module in utility and use that module here so the high level function is kept simple.

Reply:
Done!